rfc/graphql-schema/additionalTypeDefs/byPage/accountSettings.ts

```diff
@@ -0,0 +1,26 @@
+export default /* GraphQL */ `
+  type CompanyRelationship {
+    company: Company!
+    role: Role!
+  }
+
+  extend type Customer {
+    companyRelationship: CompanyRelationship
+  }
+
+  input UpdateCustomerInput {
+    firstName: String!
+    lastName: String!
+    email: String!
+    phone: String
+    formFields: CustomerFormFieldsInput
+  }
+
+  type CustomerMutations {
+    updateCustomer(input: UpdateCustomerInput!): UpdateCustomerResult
```

Comment:
This is an existing mutation/input, I added it here for completeness. If we can fold `formFields` with `extraFields` we won't need to change the mutation, just send the fields to the correct storage.